lib/broccoli-build-error.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var path = require('path');
+
+/*
+ * Subclass of `Error`.
+ *
+ * Includes detailed information about the errors from
+ * broccoli plugins, such as error type, code frames,
+ * broccoli node name and package versions.
+ */
+function BroccoliBuildError(originalError, node) {
+  var nodeJSVersion = process.version;
+
+  /*
+   * Setting versions to `null` for now as it requires
+   * adding an option to broccoli-builder to identify
+   * ember cli/ember/ember data versions.
+   */
+  this.broccoliPayload = {
+    versions: {
+      emberCLIVersion: null,
```

Comment:
also lets not camalize these.

instead:

```js
versions: {
  'broccoli-builder': ...
}
```

that way its 1:1 with what all our version stuff (package.json etc, lockfiles use)